modules/machineset-manually-scaling.adoc

```diff
@@ -7,6 +7,11 @@
 
 If you must add or remove an instance of a machine in a MachineSet, you can manually scale the MachineSet.
 
+[IMPORTANT]
+====
+You cannot scale a MachineSet up if it currently contains `0` machines.
+====
+
```

Comment:
@kalexand-rh We can scale a machineset up from `0` **manually**. 
The issue with https://bugzilla.redhat.com/show_bug.cgi?id=1732829 is the autoscaler can not scale it up from `0`. So I think we can remove the statement `You cannot scale a MachineSet up if it currently contains `0` machines.` from the machineset-manually-scaling section and only highlight it in the applying-autoscaling section.

Sorry for the turn arounds.